Javacripts/Async.js

```diff
@@ -0,0 +1,104 @@
+
```

Comment:
@codal-uchoudhary Going forward I would like to see those implementations in form of a task. You can start work on Demo React project, I have already shared a task with you. I would like to see you use this concepts wherever required in that task.